pkg/tools/mock_data/results_invalid.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
The indentation seems off in these JSON files. There's room to improve readability

Reply:
@pavannd1 As discussed offline, the indentation is off on purpose, in order to test the "invalidity" of the JSON received from the grype parser tool. Grype is an untrusted input source for this tool, because it could crash in between and dump a wrongly formatted JSON file. Please let me know if there are any better ways to testing the behavior of the parser tool when we receive bad input.